examples/Advanced_Sounding_With_Complex_Layout.py

```diff
@@ -0,0 +1,300 @@
+plt.figtext(0.71, 0.18, f'{int(mucin.m)} J/kg', weight='bold',
+            fontsize=15, color='lightblue', ha='right')
+plt.figtext(0.58, 0.13, 'TT-INDEX: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.71, 0.13, f'{int(total_totals.m)} Δ°C', weight='bold',
+            fontsize=15, color='orangered', ha='right')
+plt.figtext(0.58, 0.10, 'K-INDEX: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.71, 0.10, f'{int(kindex.m)} °C', weight='bold',
+            fontsize=15, color='orangered', ha='right')
+# now some kinematic parameters
+met_per_sec = (units.m * units.m) / (units.sec * units.sec)
+plt.figtext(0.73, 0.37, '0-1km SRH: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.37, f'{int(total_helicity1.m)* met_per_sec:~P}',
+            weight='bold', fontsize=15, color='navy', ha='right')
+plt.figtext(0.73, 0.34, '0-1km SHEAR: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.34, f'{int(bshear1.m)} kts', weight='bold',
+            fontsize=15, color='blue', ha='right')
+plt.figtext(0.73, 0.29, '0-3km SRH: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.29, f'{int(total_helicity3.m)* met_per_sec:~P}',
+            weight='bold', fontsize=15, color='navy', ha='right')
+plt.figtext(0.73, 0.26, '0-3km SHEAR: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.26, f'{int(bshear3.m)} kts', weight='bold',
+            fontsize=15, color='blue', ha='right')
+plt.figtext(0.73, 0.21, '0-6km SRH: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.21, f'{int(total_helicity6.m)* met_per_sec:~P}',
+            weight='bold', fontsize=15, color='navy', ha='right')
+plt.figtext(0.73, 0.18, '0-6km SHEAR: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.18, f'{int(bshear6.m)} kts', weight='bold',
+            fontsize=15, color='blue', ha='right')
+plt.figtext(0.73, 0.13, 'SIG TORNADO: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.13, f'{int(sig_tor[0].m)}', weight='bold', fontsize=15,
+            color='orangered', ha='right')
+plt.figtext(0.73, 0.10, 'SUPERCELL COMP: ', weight='bold', fontsize=15,
+            color='black', ha='left')
+plt.figtext(0.88, 0.10, f'{int(super_comp[0].m)}', weight='bold', fontsize=15,
+            color='orangered', ha='right')
+# add legends to the skew and hodo
+skewleg = skew.ax.legend(loc='upper left')
+hodoleg = h.ax.legend(loc='upper left')
+# add a quick plot title, this could be automated by
+# declaring a station and datetime variable when using
+# realtime observation data from siphon.
+plt.figtext(0.45, 0.97, 'OUN | MAY 4TH 1999 - 00Z VERTICAL PROFILE',
+            weight='bold', fontsize=20, ha='center')
+# Show the plot
+plt.show()
```

Comment:
Add a blank line at the end.

Reply:
Interestingly I already had a blank line at the end, and the recent commits I have just made have been trying to correct that. It shows up as a new line in my fork, but it doesn't seem to end up in the pull request. Any ideas?

(This is what it looks like in my forked repo)

![Screenshot 2023-09-26 at 13-23-59 MetPy_examples_Advanced_Sounding_With_Complex_Layout py at main · kylejgillett_MetPy](https://github.com/Unidata/MetPy/assets/100786530/7e0f2615-f255-46db-bf6d-2882568b6056)

Reply:
Hmmm, yes, I see the same thing when I pull the PR locally. @dcamron thoughts as to why we are getting this doc build failure?